src/Summoner/CLI.hs

```diff
@@ -0,0 +1,181 @@
+{-# LANGUAGE ApplicativeDo #-}
+{-# LANGUAGE QuasiQuotes   #-}
+
+-- | This module contains functions and data types to parse CLI inputs.
+
+module Summoner.CLI
+       ( summon
+       ) where
+
+import Data.Foldable (fold)
+import Data.Semigroup (Semigroup (..))
+import Data.Text (Text)
+import NeatInterpolation (text)
+import Options.Applicative (Parser, ParserInfo, command, execParser, flag, fullDesc, help, helper,
+                            info, infoFooter, infoHeader, long, metavar, optional, progDesc, short,
+                            strArgument, subparser)
+import Options.Applicative.Help.Chunk (stringChunk)
+
+import Summoner.Ansi (bold, reset)
+import Summoner.Default (endLine)
+import Summoner.Project (Decision (..), Targets (..), generateProject)
+
+import qualified Data.Text as T
+import qualified Data.Text.IO as T
+
+----------------------------------------------------------------------------
+-- CLI
+----------------------------------------------------------------------------
+
+summon :: IO ()
+summon = execParser prsr >>= runWithOptions
+
+-- | Run 'hs-init' with cli options
+runWithOptions :: InitOpts -> IO ()
+runWithOptions (InitOpts projectName targets) = do
+     -- Generate the project.
+    generateProject projectName targets
+
+    bold
+    T.putStrLn "\nJob's done"
```

Comment:
This looks similar to `boldDefault` function. Maybe you can create separate helper function for this to slightly remove code boilerplate?